dynamiqs/__init__.py

```diff
@@ -1,4 +1,5 @@
 from . import dark, plot, random
+from .hermitian_conjugate import hc
```

Comment:
For imports throughout the library, we use `from .x import *` and define a `__all__ = [...]` in the file. This structure is needed for documentation afterwards.